src/components/BaseMap/BaseMap.tsx

```
@@ -224,46 +329,33 @@ export default function BaseMap({
-                      'line-color': 'black',
-                    }}
-                  />
-                </Source>
-              )
-            : isMapLoaded && layer.isLayerOn && (
-                <Source
-                  id={layer.sourceId}
-                  key={`${layer.sourceId}-${index}`}
+          return layer.dataType === 'pmtiles' ? (
+            isMapLoaded && layer.layerId === 'watershed' ? (
+              <WatershedLayers layer={layer} index={index} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)